env.default

```diff
@@ -40,7 +40,7 @@ CSP_CONNECT_SRC="*"
 CSP_DEFAULT_SRC="'none'"
 CSP_FONT_SRC="'self' https://fonts.gstatic.com https://fonts.googleapis.com https://code.cdn.mozilla.net"
 CSP_FRAME_ANCESTORS="'none'"
-CSP_FRAME_SRC="'self' https://www.youtube.com https://comments.mozillafoundation.org/ https://airtable.com https://docs.google.com/ https://platform.twitter.com https://public.zenkit.com https://calendar.google.com https://www.youtube-nocookie.com https://devopstypeform.typeform.com"
+CSP_FRAME_SRC="'self' https://www.youtube.com https://comments.mozillafoundation.org/ https://airtable.com https://docs.google.com/ https://platform.twitter.com https://public.zenkit.com https://calendar.google.com https://www.youtube-nocookie.com https://devopstypeform.typeform.com  https://form.typeform.com"
```

Comment:
```suggestion
CSP_FRAME_SRC="'self' https://www.youtube.com https://comments.mozillafoundation.org/ https://airtable.com https://docs.google.com/ https://platform.twitter.com https://public.zenkit.com https://calendar.google.com https://www.youtube-nocookie.com https://form.typeform.com"
```

Reply:
Seems like we no longer need to whitelist `https://devopstypeform.typeform.com` for the embed to work